PF/Assignment4/Permutation.java

```diff
@@ -0,0 +1,25 @@
+import java.util.List;
+
+public class Permutation {
+
+	public static List<String> permuteString(String beginningString,
```

Comment:
The simplest thing you can do is 'permuteString()' function be a private function and the function which is asked in the problem be the public one and in this public function you can call your private function as you want.